Initium-Odp/src/com/universeprojects/miniup/server/scripting/jsaccessors/DBAccessor.java

```diff
@@ -322,6 +322,15 @@ public boolean unequipItem(EntityWrapper item, EntityWrapper currentCharacter)
 		return removed;
 	}
 	
+	/**
+	 * Sends a game message to the specified characters
+	 */
+	public void sendMessageToCharacters(String message, Character... characters) {
+		for(Character target:characters) {
+			db.sendGameMessage(db.getDB(), target.wrappedEntity, message);
+		}
+	}
+	
```

Comment:
Can't do this. If an error occurs during the script, then these calls shouldn't happen. You need to queue up any game messages you want to send, and only send when the call was successful. Look at `ScriptEvent` for `sendGameUpdates` and `CommandScriptBase` for `!event.haltExecution` code path.